imod/protocols/protocol_tomoReconstruction.py

```diff
@@ -356,24 +364,31 @@ def createOutputStep(self, tsId):
 
                 # Set default tomogram origin
                 newTomogram.setOrigin(newOrigin=None)
-                if self.tomoShiftZ.get():
+                shiftX = self.tomoShiftX.get()
+                shiftZ = self.tomoShiftZ.get()
+                if shiftX or shiftZ:
+                    sRate = newTomogram.getSamplingRate()
                     x, y, z = newTomogram.getShiftsFromOrigin()
-                    shiftZang = self.tomoShiftZ.get() * newTomogram.getSamplingRate()
-                    newTomogram.setShiftsInOrigin(x=x, y=y, z=z + shiftZang)
+                    shiftXang = shiftX * sRate
+                    shiftZang = shiftZ * sRate
+                    newTomogram.setShiftsInOrigin(x=x - shiftXang, y=y, z=z - shiftZang)
 
                 output.append(newTomogram)
-                output.updateDim()
                 output.update(newTomogram)
-
                 output.write()
                 self._store(output)
+            else:
```

Comment:
I can see you put back these two:
output.write()
self._store(output)
Is there a reason? Protocol works without these redundant calls. Write and store are called in the closeOutputSet step for all outputs